.github/workflows/build-test-lint.yaml

```diff
@@ -16,26 +16,21 @@ on:
 
 jobs:
   build:
-    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.1
+    uses: scientist-softserv/actions/.github/workflows/build.yaml@v0.0.4
+    secrets: inherit
     with:
       target: hyku-base
       worker: true
       workerTarget: hyku-worker
-      # tag: deca90d9
   test:
     needs: build
-    uses: scientist-softserv/actions/.github/workflows/test.yaml@v0.0.1
+    uses: scientist-softserv/actions/.github/workflows/test.yaml@v0.0.3
```

Comment:
`test.yaml@v0.0.4` should work now; see https://github.com/scientist-softserv/actions/pull/15